Update automation-levels.md (#12555)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/automation-levels.md

- If your security team has defined device groups with a level of automation, those settings aren't changed by the new default settings that are rolling out.

- You can keep your default automation settings, or change them according to your organizational needs. To change your settings, [set your level of automation](/microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation#set-up-device-groups).

> - > [!NOTE]
>   > Defender for Business depents on realtime protection for Automatic invetigation , so Defender for Business should be running in active mode to have the Automatic invetigation 

Reviewer: ```suggestion

> [!NOTE]
> Defender for Business depends on real-time protection for automatic investigation, so it should be running in active mode to enable automatic investigation.
```

Author: Changes look good to me, Thanks
